From b506c435ffe028e73077205c9ee61ca17f8c0529 Mon Sep 17 00:00:00 2001
From: Hans Yang <yhx@rock-chips.com>
Date: Thu, 12 Nov 2020 17:13:39 +0800
Subject: [PATCH] arm64: configs: add rockchip_linux_bifrost.config

for linux platform bifrost can not compatible with mali config

Update by:
make ARCH=arm64 rockchip_linux_defconfig
cp .config rockchip_linux.config
make ARCH=arm64 menuconfig
scripts/diffconfig -m rockchip_linux.config .config >
arch/arm64/configs/rockchip_linux_bifrost.config

Usage:
make ARCH=arm64 rockchip_linux_defconfig rockchip_linux_bifrost.config

Signed-off-by: Hans Yang <yhx@rock-chips.com>
Change-Id: Ic5def57928c631c96754bb4dd76404d8cbd1e2e6
---
 arch/arm64/configs/rockchip_linux_bifrost.config | 12 ++++++++++++
 1 file changed, 12 insertions(+)
 create mode 100644 arch/arm64/configs/rockchip_linux_bifrost.config

diff --git a/arch/arm64/configs/rockchip_linux_bifrost.config b/arch/arm64/configs/rockchip_linux_bifrost.config
new file mode 100644
index 000000000000..1eac5c14ff91
--- /dev/null
+++ b/arch/arm64/configs/rockchip_linux_bifrost.config
@@ -0,0 +1,12 @@
+CONFIG_MALI_BIFROST=y
+# CONFIG_MALI_MIDGARD is not set
+# CONFIG_MALI_ARBITER_SUPPORT is not set
+CONFIG_MALI_BIFROST_DEVFREQ=y
+# CONFIG_MALI_BIFROST_DMA_FENCE is not set
+# CONFIG_MALI_BIFROST_ENABLE_TRACE is not set
+# CONFIG_MALI_BIFROST_EXPERT is not set
+# CONFIG_MALI_BIFROST_GATOR_SUPPORT is not set
+# CONFIG_MALI_DMA_BUF_LEGACY_COMPAT is not set
+# CONFIG_MALI_DMA_BUF_MAP_ON_DEMAND is not set
+# CONFIG_MALI_GEM5_BUILD is not set
+CONFIG_MALI_PLATFORM_NAME="rk"
-- 
2.35.3

